File: Lab6/src/test/java/com/example/lab6/data/DataBaseConnectorTest.java
package com.example.lab6.data;

import org.junit.jupiter.api.Test;

import java.sql.SQLException;

import static org.junit.jupiter.api.Assertions.*;

class DataBaseConnectorTest {

    @Test
    void testDriver() {
        DataBaseConnector testConnector =
                new DataBaseConnector("testDB");
        assertTrue(testConnector.testDriver());
    }

    @Test
    void getConnection() {
        DataBaseConnector testConnector =
                new DataBaseConnector("testDB");
        try {
            assertNotNull(testConnector.getConnection());
        } catch (SQLException e) {
            fail();
        }
    }
}

File: Lab6kotlin/src/test/java/com/example/lab6/data/DataBaseConnectorTest.kt
package com.example.lab6.data

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test
import java.sql.SQLException

class DataBaseConnectorTest {

    @Test
    fun testDriver() {
        val testConnector = DataBaseConnector("testDB")
        assertTrue(testConnector.testDriver())
    }

    @Test
    fun getConnection() {
        val testConnector = DataBaseConnector("testDB")
        try {
            Assertions.assertNotNull(testConnector.connection)
        } catch (e: SQLException) {
            Assertions.fail<Any>()
        }
    }
}